pimpmycause/pimpmycause/templates/registration/registration_complete.html

```diff
@@ -3,8 +3,8 @@
 {% load static %}
 
 {% block main %} 
-	<div class="pnl-title page-register">
-		<h1>Thanks!</h1>
+	<div class="page-register pnl">
+		<h1 class="pnl-title">Thanks!</h1>
```

Comment:
adding in pnl because this class had the styles needed for a pnl to have the right padding/margin. So pnl class in like a section. So it was not applied on any of the page sections so far but homepage.